docs/user/openstack/install_upi.md

```diff
@@ -79,6 +76,37 @@ $ mkdir -p openstack-upi
 $ cd openstack-upi
 ```
 
+### Red Hat Enterprise Linux CoreOS (RHCOS)
+
+Get the latest RHCOS image [here](https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/latest/). You must download images with the highest version that is less than or equal to the OpenShift version that you install. Use the image versions that match your OpenShift version if they are available.
+
+**NOTE:** The OpenStack QCOW2 image is delivered in compressed format.  You will need to provide the necessary options to `curl` or `wget` when downloading the image, otherwise your image may appear corrupted.
+For example:
+
+```sh
+curl --compressed -J -L -O <url of OpenStack QCOW2>
+```
+
+or
+
+```sh
+wget --compression=auto <url of OpenStack QCOW2>
+```
```

Comment:
This isn't true anymore. The downloaded file now needs to be uncompressed with `gzip`, and soon with `xz`.

Reply:
Yeah, it's not necessary now